.github/workflows/samples-kotlin-wiremock.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
       matrix:
         sample:
           - samples/server/petstore/kotlin-wiremock
+          - samples/server/petstore/kotlin-wiremock-responses
```

Comment:
i think you will need to update line 6 to 
```
      - samples/server/petstore/kotlin-wiremock*/**
```

Reply:
Agreed, fix is pushed

Reply:
👍 

will merge when all tests pass